index.js

```diff
@@ -62,3 +62,14 @@ module.exports.buffer = (input, opts) => {
 
 	return pipe.then(buf => buf.length < input.length ? buf : input);
 };
+
+module.exports.stats = (max, min) => {
+	const difference = Math.abs(max.length - min.length);
+
+	return {
+		difference,
+		max: max.length,
+		min: min.length,
+		percent: (difference / max.length) * 100
```

Comment:
We could also provide a default message implementations can use: https://github.com/sindresorhus/gulp-imagemin/blob/d933de19aaa790b567cf724619aa39b975c71280/index.js#L64-L65